src/components/Scheme/ContributionRewardExtRewarders/README.md

```diff
@@ -0,0 +1,5 @@
+# ContributionRewardExt Rewarder Registry
```

Comment:
So I believe we talked about this, but for the record, I do not think this is necessary, and I also think it is not worth the effort. 

I believe we all  (me, adam, oren, you?) think the "rewarder" pattern used in ContributionRewardExt.sol` was a bad idea, so it is not likely that we are going to re-use it in the future. So it is a wasted effort to try to anticipate other uses (and it does make the code more complex...).

Not sure how finished this modularity part is, but if it is more work to finish this then to remove it, I vote strongly for removing it

Reply:
The work is 100% finished.  It works.

The current architecture may have been portrayed as a "bad idea", but at this time it is what we have and I don't see it being rewritten any time soon.  The architecture works and no decision has been made to change it.  So I think we maintain optionality by keeping this code, staying ready in case someone does want to add a new rewarder contract, which will be really easy to do.

Reply:
:+1: